app/Http/Controllers/API/VoucherController.php

```diff
@@ -84,7 +80,17 @@ function($dirtycode) {
         $failed_rejects = [];
         $vouchers_for_payment = [];
 
+        if ($transition == 'confirm') {
+            // Tidy the StateTokens
+            StateToken::tidy();
+
+            // We'll need a StateToken for Later
+            $stateToken = new StateToken();
```

Comment:
Hey - did you ever figure out the tinker crash problem?

Reply:
Yes! You cant call a static database function belonging to that class from inside the constructor's scope, apparently.

Reply:
Static DB functions are bad anyway. :wink: